b2share/modules/communities/errors.py

```diff
@@ -39,3 +39,7 @@ class CommunityDoesNotExistError(Exception):
 class CommunityDeletedError(Exception):
     """Exception raised when a requested community is marked as deleted."""
     pass
+
+class GetAllStartStopError(Exception):
```

Comment:
I'd say this error is too specific, how about one named InvalidArgumentsError? A more general, reusable error may be more useful.